src/main/java/edu/ucsb/cs156/rec/controllers/DummyController.java

```diff
@@ -0,0 +1,26 @@
+package edu.ucsb.cs156.rec.controllers;
```

Comment:
Please move `src/main/java/edu/ucsb/cs156/rec/controllers/DummyController.java` to 
`src/test/java/edu/ucsb/cs156/rec/controllers/DummyController.java`.

It is not a real controller, and exists only to help with test coverage.  So it should not be under `/src/main/java` but under `/src/test/java`.  That way it doesn't really show in the real app; only when testing.